Documentation/WritingWithInk.md

```diff
@@ -1820,7 +1820,7 @@ And sometimes the numbers are useful in other ways:
 	VAR suitcase_location = HALLWAY
 	
 	=== report_progress ===
-	{  secret_agent_location = suitcase_location:
+	{  secret_agent_location == suitcase_location:
```

Comment:
:+1: This produces: `ERROR: ... Expected some kind of logic, conditional or sequence within braces: { ... } but saw ...`